client/static/js/index.js

```diff
@@ -0,0 +1,83 @@
+const searchButton = document.getElementById('search');
+const surpriseButton = document.getElementById('surprise');
+const searchBar = document.getElementById('searchBar');
+const searchEngine = document.getElementById('searchEngine')
+const searchResults = document.getElementById('results');
+const logo = document.getElementById('logo');
+
+//function to show 10 search results
+function getSearchResult(e) {
+    //handling spaces in searches
+    const searchString = searchBar.value.replace(/\s/g, "_");
+
+    //hiding logo and showing results
+    searchEngine.className = "moveUp";
+    logo.className = "hide";
+    searchResults.className = "";
+
+    fetch(`http://localhost:3000/${searchString}`)
+       .then(resp => resp.json())
+       .then(resp => { 
+            if(resp.error) {
+                //displays message if no results are found
+                const noResults = document.createElement("H1");
+                const noResultsMessage = document.createTextNode(`No results found for \"${searchString}\".`);
+                noResults.appendChild(noResultsMessage);
+                searchResults.appendChild(noResults);
+            }
+
+            for(let i = 0; i < resp.length; i++) //loop through all array elements
+            {
+                //adding website name to search results
+                const websiteName = document.createElement("A");
+                const websiteNameText = document.createTextNode(`${resp[i].header}`);
+                websiteName.setAttribute("href", `${resp[i].link}`);
+                websiteName.appendChild(websiteNameText);
+                searchResults.appendChild(websiteName);
+                
+                //adding website info to search results
+                const websiteInfo = document.createElement("P");
+                const websiteInfoText = document.createTextNode(`${resp[i].bodyText}`);
+                websiteInfo.appendChild(websiteInfoText);
+                searchResults.appendChild(websiteInfo);
+            };
+        })
+        .catch(err => { 
+            console.log(err) });
+}
+
```

Comment:
when you search something on the homepage it gives u a limited amount items, but when you search dogs on the results page it gives us every single result it can find.